contracts/escrow.js

```diff
@@ -2,7 +2,7 @@ import { call, LookupMap, NearBindgen, view, assert, near, UnorderedMap, NearPro
 
 @NearBindgen({})
 export class EscrowContract {
-  GAS_FEE = 30_000_000_000_000; // 30 TGAS
+  // GAS_FEE = 300_000_000_000_000; // 300 TGAS
```

Comment:
Why this change if all calls below remain at 30TGas?

Reply:
I just fixed this and changed the default to 30TGas while repassing the TGas variable. One question I have regarding the parent code is what you were trying to achieve in this [line of code](https://github.com/near-examples/escrow-js/blob/4929ec4c29fd54e9c1cea523940dc922afacd723/contracts/escrow.js#L44)

Reply:
add readability to the repo, instead of passing the fix amount every time